src/appleseed.studio/mainwindow/rendering/renderingmanager.h

```diff
@@ -202,6 +202,13 @@ class RenderingManager
     void slot_camera_change_begin();
     void slot_camera_changed();
     void slot_camera_change_end();
+    void slot_material_dropped(
+        const foundation::Vector2d& drop_pos,
+        const std::string& material_name);
```

Comment:
Align arguments vertically (check how it's done throughout the codebase).

Reply:
Add any necessary header file (I haven't checked if you need anything new, but just in case).